xbmc/pvr/timers/PVRTimers.cpp

```diff
@@ -521,7 +521,7 @@ bool CPVRTimers::InstantTimer(const CPVRChannelPtr &channel)
   {
     newTimer.reset(new CPVRTimerInfoTag);
     /* set the timer data */
-    newTimer->m_iClientIndex      = -1;
+    newTimer->m_iClientIndex      = PVR_INVALID_TIMER_ID;
```

Comment:
What about the other places where m_iClientIndex is assigned -1. Do a grep, man!